pandas/tseries/tests/test_timeseries.py

```diff
@@ -1797,7 +1800,10 @@ def test_append_concat_tz_explicit_pytz(self):
     def test_append_concat_tz_dateutil(self):
         # GH 2938
         tm._skip_if_no_dateutil()
-        from dateutil.zoneinfo import gettz as timezone
+        if sys.platform != 'win32':
```

Comment:
except this test as its different (you could also do something like ``tslib._dateutil_timezone`` like you do for gettz)